src/components/NavColab.vue

```diff
@@ -111,6 +111,7 @@ export default {
       }
       colabText.value = 'Open in Colab'
     }
+    // watch(store.config, () => downloadProject)
```

Comment:
Do we need this ? In general we should remove commented code or explain why it is in the codebase

Reply:
Ahh, It's not required there. Didn't mean to commit that change...